Nick_Maven/WebdriverAdv/Yandex/POModel/BrowserParams.java

```diff
@@ -0,0 +1,35 @@
+package Nick_Maven.WebdriverAdv.Yandex.POModel;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.ArrayList;
+
+public class BrowserParams {
+    private WebDriver driver;
+    private ArrayList<String> tabs;
+
+    public BrowserParams(WebDriver driver) {
+        this.driver = driver;
+        driver.manage().window().maximize();
+        PageFactory.initElements(driver, this);
+    }
+    public BrowserParams switchZeroTab() {
+        driver.switchTo().window(tabs.get(0));
+        return this;
+    }
+    public BrowserParams switchFirstTab() {
+        driver.switchTo().window(tabs.get(1));
+        return this;
+    }
+    public BrowserParams switchSecondTab() {
+        driver.switchTo().window(tabs.get(2));
+        return this;
+    }
```

Comment:
Why do you create three methods for the same functionality?
public BrowserParams switchToTab(int tabNumber) {
        driver.switchTo().window(tabs.get(tabNumber));
        return this;
    }

Reply:
still was not fixed